docs/application/tizen-studio/release-notes/3-6-release-notes.md

```diff
@@ -0,0 +1,59 @@
+# Tizen Studio 3.6 Release Notes
+
+- Release Date: Nov 30, 2019
```

Comment:
- Release Date: Nov 30, 2019

Will Tizen Studio 3.6 really be published on Nov. 30?
Or is Nov 30. the temp value?

If Tizen Studio 3.6 will be published on other date,
I think it would be better to update this line to actual date.

Reply:
The current temp date is 30Nov , RC is already in progress , if the release is delayed i will update the exact date on confirmation from SQA